staging/src/k8s.io/pod-security-admission/test/fixtures_seccomp_baseline.go

```diff
@@ -0,0 +1,136 @@
+		generatePass: func(p *corev1.Pod) []*corev1.Pod {
+			if val, ok := p.Annotations[annotationKeyPod]; ok && val == corev1.SeccompProfileRuntimeDefault {
+				return []*corev1.Pod{p}
+			}
+
+			p = ensureAnnotation(p)
+			return []*corev1.Pod{
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyPod] = corev1.SeccompProfileRuntimeDefault
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyPod] = corev1.SeccompLocalhostProfileNamePrefix + "testing"
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyContainer(p.Spec.Containers[0])] = corev1.SeccompProfileRuntimeDefault
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyContainer(p.Spec.Containers[0])] = corev1.SeccompLocalhostProfileNamePrefix + "testing"
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyContainer(p.Spec.InitContainers[0])] = corev1.SeccompProfileRuntimeDefault
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyContainer(p.Spec.InitContainers[0])] = corev1.SeccompLocalhostProfileNamePrefix + "testing"
+				}),
+			}
+		},
+		generateFail: func(p *corev1.Pod) []*corev1.Pod {
+			p = ensureAnnotation(p)
+			return []*corev1.Pod{
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyPod] = corev1.SeccompProfileNameUnconfined
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyContainer(p.Spec.Containers[0])] = corev1.SeccompProfileNameUnconfined
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Annotations[annotationKeyContainer(p.Spec.InitContainers[0])] = corev1.SeccompProfileNameUnconfined
+				}),
+			}
+		},
+	}
+
+	fixtureData_baseline_1_19 := fixtureGenerator{
+		expectErrorSubstring: "seccomp profile",
+		generatePass: func(p *corev1.Pod) []*corev1.Pod {
+			if p.Spec.SecurityContext != nil {
+				if p.Spec.SecurityContext.SeccompProfile != nil {
+					if p.Spec.SecurityContext.SeccompProfile.Type == corev1.SeccompProfileTypeRuntimeDefault {
+						return []*corev1.Pod{p}
+					}
+				}
+			}
+
+			p = ensureSecurityContext(p)
+			return []*corev1.Pod{
+				p,
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.SecurityContext.SeccompProfile = seccompProfileRuntimeDefault
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.SecurityContext.SeccompProfile = seccompProfileLocalhost("testing")
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.Containers[0].SecurityContext.SeccompProfile = seccompProfileRuntimeDefault
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.Containers[0].SecurityContext.SeccompProfile = seccompProfileLocalhost("testing")
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.InitContainers[0].SecurityContext.SeccompProfile = seccompProfileRuntimeDefault
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.InitContainers[0].SecurityContext.SeccompProfile = seccompProfileLocalhost("testing")
+				}),
```

Comment:
same as above, two passing testcases, one which populates pod/container/initcontainer with the default profile, and one which populates with the localhost profile